packages/odyssey/src/scss/abstracts/_colors.scss

```diff
@@ -17,66 +17,63 @@
 // Color Variants
 
 $colors: (
-  'success': (
-    'lightest': #e5fff6,
-    'base': #00d1b3,
-    'dark': #00a08e,
-  ),
-  'action': (
-    'lightest': #e5edfb,
-    'base': #1662dd,
-    'dark': #124a94,
+  'gray': (
+    '000': #f5f5f6,
+    '100': #ebebed,
+    '200': #d7d7dc,
+    '300': #c1c1c8,
+    '500': #aaaab4, // Temporary, required for Admin Demo
+    '400': #aaaab4,
+    '600': #6e6e78,
+    '700': #212126, // Temporary, required for Admin Demo
+    '800': #212126, // Temporary, required for Admin Demo
+    '900': #212126,
   ),
-  'caution': (
-    'lightest': #fcf9e1,
-    'base': #ebd551,
-    'dark': #bc960f,
+  'blue': (
+    '000': #f2f5ff,
+    '300': #a7b5ec,
+    '500': #1662dd,
+    '900': #00297a,
   ),
-  'warning': (
-    'lightest': #feefe2,
-    'base': #f47c25,
-    'dark': #cb5500,
+  'green': (
+    '000': #e5faf3,
+    '300': #84d2b1,
+    '500': #00b478,
+    '900': #00503c,
   ),
-  'danger': (
-    'lightest': #fee6ea,
-    'base': #dd0744,
-    'dark': #b80047,
+  'red': (
+    '000': #ffdfe2,
+    '300': #f88c90,
+    '500': #da372c,
+    '900': #640019,
   ),
-  'cyan': (
-    'lightest': #c6fafa,
-    'base': #007f9c,
-    'dark': #004d6b,
+  // caution
+  'yellow': (
+    '000': #faf5dc,
+    '300': #f9dc77,
+    '500': #ffc61c,
+    '900': #663800,
   ),
-  'violet': (
-    'lightest': #f3eafe,
-    'base': #963fe6,
-    'dark': #5a13b4,
+  'orange': (
+    '500': #e97107
   ),
-  'magenta': (
-    'base': #c72c86,
+  'turquoise': (
+    '500': #2d8c9e
   ),
-  'gray': (
-    '000': #fafafa,
-    '100': #e4e5e7,
-    '200': #cdd1d3,
-    '300': #b7bcc0,
-    '400': #a0a7ac,
-    '500': #899298,
-    '600': #737d85,
-    '700': #5c6971,
-    '800': #46545e,
-    '900': #2f3f4a,
+  'purple': (
+    '500': #8d6e97
   ),
 );
 
+$black: #000000;
 $white: #ffffff;
 
 /**
  * Retrieves color variants from variation map.
  * color: cv(action, light); // color: #96bafa;
  */
 
-@function cv($color, $variation: 'base') {
+@function cv($color, $variation: '500') {
```

Comment:
With our hackathon we ran into issues with SCSS functions when building our generator for native components. Most utilities out there require tokenized values instead of retrieving them via a variation map.

What advantage does this have over providing actual HEX/RGB colors for these variants? IMO - it's much cleaner (and removes the need to confirm color lookup) to reference the variables instead of passing in a function.

Reply:
We can iterate over maps, which makes them super useful for internal use. We're going to be revisiting tokens after this release to ensure we have a full set of tokens such that our `cv()` and `ms()` functions won't be required for normal use.